rules/windows/credential_access_spn_attribute_modified.toml

```diff
@@ -0,0 +1,126 @@
+[metadata]
+creation_date = "2022/02/22"
+maturity = "production"
+updated_date = "2022/02/22"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects when a user account has the servicePrincipalName attribute modified. Attackers can abuse write privileges over a
+user to configure SPNs so that they can perform Kerberoasting. Administrators can also configure this for legitimate
+purposes, exposing the account to Kerberoasting.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-system.*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "User account exposed to Kerberoasting"
```

Comment:
```suggestion
name = "User account exposed to Targeted Kerberoasting"
```

Reply:
This looks for the misconfiguration, as targeted Kerberoasting would remove this right after requesting the ticket